donkeycar/tests/test_tub_v2.py

```diff
@@ -7,11 +7,12 @@
 
 class TestTub(unittest.TestCase):
 
-    def setUp(self):
-        self._path = tempfile.mkdtemp()
+    @classmethod
+    def setUpClass(cls) -> None:
```

Comment:
Out of curiosity, why did you convert this to a `@classmethod` and not leave it as-is. Not that it matters, but I just wanted to understand if there is a reason.

Reply:
Because `setUp`  and `tearDown` are called before/after each test function in the class. But here I want to keep the tub alive, because we fill it in the first test method and then use it in the second. `setUpClass` and `tearDownClass` are only called before the first test method and after the last one.